llvm/test/DebugInfo/Sparc/pointer-add-unknown-offset-debug-info.ll

```diff
@@ -12,7 +12,7 @@ define void @pointer_add_unknown_offset(ptr %base, i32 %offset) !dbg !7 {
   ; CHECK-NEXT:   [[COPY:%[0-9]+]]:i64regs = COPY $i1
   ; CHECK-NEXT:   [[COPY1:%[0-9]+]]:i64regs = COPY $i0
   ; CHECK-NEXT:   [[SRAri:%[0-9]+]]:i64regs = SRAri [[COPY]], 0
-  ; CHECK-NEXT:   [[SLLXri:%[0-9]+]]:i64regs = SLLXri killed [[SRAri]], 2
+  ; CHECK-NEXT:   [[SLLXri:%[0-9]+]]:i64regs = nsw SLLXri killed [[SRAri]], 2
```

Comment:
@arsenm  - is the change in this PR supposed to be specific to this target?

Reply:
No. It just introduces the flags, which may or may not survive to the end of the target's selection

Reply:
Could you please come up with a test for another Target, say for example AArch64?

Reply:
This test change is incidental, this test was not the point. The real tests are supposed to be from 671cbcf6420a9dfc81754d8dc683619f6f396ed8 (but in the final version of the patch, the codegen changes no longer manifest)